package.json

```diff
@@ -49,7 +49,8 @@
     "**/@roadiehq/**/@backstage/plugin-catalog": "*",
     "**/@roadiehq/**/@backstage/catalog-model": "*",
     "graphql-language-service-interface": "2.8.2",
-    "graphql-language-service-parser": "1.9.0"
+    "graphql-language-service-parser": "1.9.0",
+    "lodash": "^4.17.21"
```

Comment:
Tbh I'm not sure about remedying vulnerabilities using resolutions in the main repo. It doesn't solve the issue for anyone using our packages and risks hiding that problem from us. I think bumping our dependencies and then striving to find ways to properly get rid of the vulnerabilities is better, and otherwise let them be and recommend that people use resolutions to get around it themselves.